WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -3,6 +3,10 @@
 <div class="container">
   <h2><%= yield(:title) %></h2>
 
+  <div style="background-color: #ff0000 ; padding: 5px; border: 1px solid red; color:white;">
+    <h4><%= t('.disclaimer') %></h4>
+  </div>
```

Comment:
I think we can re-use the red alert we use in other places, so:

```erb
<%= alert :danger, t('.disclaimer') %>
```

Let me know if that looks appropriate =)

Reply:
That would make sense I think. Just pushed that change!